src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetOutstandingIpos/GetOutstandingIposForCurrentPersonQueryHandlerTests.cs

```diff
@@ -694,6 +693,54 @@ public async Task Handle_ShouldNotReturnIpoForConstructionCompanyFunctionalRole_
             }
         }
 
+        [TestMethod]
+        public async Task Handle_ShouldNotReturnIpoForConstructionCompanyFunctionalRole_AfterIpoScopeHasBeenHandedOver()
```

Comment:
I find this test very hard to read. 

Very long names, no arrange act assert, and no helping comments or assert messages.

outstandingInvitationWithPersonParticipantContractor -> personPatricipant etc?

Also, do we need 2 using statements and contexts here?

 context.SaveChangesAsync().Wait() -> await context.SaveChangesAsync().

using -> await using

what is behind result.Data.Items.ElementAt(4)?